.travis.yml

```diff
@@ -67,15 +68,17 @@ script:
       bumpversion --new-version=1.$TRAVIS_JOB_NUMBER minor
       git push git@github.com:ionelmc/python-nameless.git -f HEAD:${PR}test-$ENV
     else
-      echo Failed to get publish-key
+      echo Failed to get publish-key, running tests locally...
+      set -x
```

Comment:
Oh! So we can actually see what it's doing in the complex command! I am totally stealing this for the Docker build.